include/circt/Dialect/HW/HWAttributes.td

```diff
@@ -11,14 +11,75 @@
 //===----------------------------------------------------------------------===//
 
 /// An attribute to indicate the output file an operation should be emitted to.
-def OutputFileAttr : StructAttr<"OutputFileAttr", HWDialect, [
-  StructFieldAttr<"directory", OptionalAttr<StrAttr>>,
-  StructFieldAttr<"name", OptionalAttr<StrAttr>>,
-  StructFieldAttr<"exclude_from_filelist",
-    DefaultValuedAttr<BoolAttr, "false">>,
-  StructFieldAttr<"exclude_replicated_ops",
-    DefaultValuedAttr<BoolAttr, "true">>,
-]>;
+def OutputFileAttr : AttrDef<HWDialect, "OutputFile"> {
```

Comment:
It looks like this commit included the OutputFileAttr changes as part of them?  In the future, it would be great to split unrelated changes like this into their own separate pushes.

Reply:
I've had good luck in the past with github connecting the PRs together properly, but I guess since I used the rebase option all 3 commits are connected to this PR.  Github did auto-close the other PR though :S